### PR TITLE
Nav changes and styling changes for subheadings in discover section

### PR DESCRIPTION
**Pull request by allisonnorman**
- Changed stakeholder file name from "process" to "stakeholders"
- Updated navigation titles for discover section
- Updated md styling from bold to ## for each subheading in the discover section

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -5,10 +5,10 @@ primary:
     links:
       - text: Overview
         href: /discover/
-      - text: Understand the affected users and stakeholders
+      - text: Understand the users and their 'as is' process
         href: /discover/audience/
```
Comment (colinmurphy01) [inline _data/navigation.yml]: consider updating
```suggestion
        href: /discover/users/
```

File: discover/audience.md [hunk 2]
```diff
@@ -44,15 +44,15 @@ Your partners likely have an incomplete view of the current state or what their
...
+## Incorporation: What to do next
 
 - Report out to organizational stakeholders to build empathy around the current state.
 
 - Translate your findings into preliminary recommendations or decision points around the problems and use cases to support.
 
 - Draw on your research to help your partner articulate a concise problem statement, create an inspirational vision, and develop a strategy for achieving their desired [future state]({{site.baseurl}}/future/).
 
-**RESOURCES**
+## RESOURCES
```
Comment (colinmurphy01) [inline discover/audience.md]:
```suggestion
## Resources
```

File: discover/stakeholders.md [hunk 3]
```diff
@@ -32,15 +32,15 @@ Sometimes your partner will have a good understanding of who these stakeholders
...
+## Incorporation: What to do next
 
 - Draw on what you’ve learned from your stakeholders to refine your initial stakeholder map.
 
 - Work with your partner to create a stakeholder engagement plan that tailors the content and frequency of your [communications]({{site.baseurl}}/deliver/collaborate) to their needs and interests.
 
 - Incorporate your stakeholders’ needs and interests when defining the [future state]({{site.baseurl}}/future/).
 
-**RESOURCES**
+## RESOURCES
```
Comment (colinmurphy01) [inline discover/stakeholders.md]:
```suggestion
## Resources
```

File: discover/technical.md [hunk 4]
```diff
@@ -33,15 +33,15 @@ Technical discovery will typically be led by engineering, but product managers s
...
+## Incorporation: What to do next
 
 - Report out on assessment in order to facilitate a decision amongst stakeholders on the best available solutions.
 
 - If gaps in knowledge or confidence exist, timebox a quick prototype (ideally with a real use case or data) to further assess technical feasibility.
 
 - Draw on your research to develop a technical roadmap that will help your partner reach their desired [future state]({{site.baseurl}}/future/).
 
-**RESOURCES**
+## RESOURCES
```
Comment (colinmurphy01) [inline discover/technical.md]:
```suggestion
## Resources
```

**Review by colinmurphy01 — CHANGES_REQUESTED** [review]
Added some comments and suggestions.

I couldn't do this within the review UI, but additionally can you update the page titles to sentence case.

**Comment by allisonnorman** [comment]
@colinmurphy01 Thanks! I made your changes and also updated the file name for /users to reflect your url path naming suggestion.

**Review by colinmurphy01 — APPROVED** [review]

**Pull request merged**